[CI] DeprecationHttpIT testDeprecationIndexingCacheReset failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #4480 / windows-2022_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/gy33ob6gkgp2w)
- [elasticsearch-periodic-platform-support #4420 / windows-2019_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/7pzm5p4znqv54)

**Reproduction Line:**
```
gradlew ":x-pack:plugin:deprecation:qa:rest:javaRestTest" --tests "org.elasticsearch.xpack.deprecation.DeprecationHttpIT.testDeprecationIndexingCacheReset" -Dtests.seed=9A57E23B4E1B6CF1 -Dtests.locale=am-ET -Dtests.timezone=Asia/Kuching -Druntime.java=22
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('testDeprecationIndexingCacheReset'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.deprecation.DeprecationHttpIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: org.elasticsearch.client.ResponseException: method [GET], host [http://127.0.0.1:58450], URI [/.logs-deprecation.elasticsearch-default/_search?q=elasticsearch.http.request.x_opaque_id:testDeprecationIndexingCacheReset], status line [HTTP/1.1 404 Not Found]
{"error":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index [.logs-deprecation.elasticsearch-default]","resource.type":"index_or_alias","resource.id":".logs-deprecation.elasticsearch-default","index_uuid":"_na_","index":".logs-deprecation.elasticsearch-default"}],"type":"index_not_found_exception","reason":"no such index [.logs-deprecation.elasticsearch-default]","resource.type":"index_or_alias","resource.id":".logs-deprecation.elasticsearch-default","index_uuid":"_na_","index":".logs-deprecation.elasticsearch-default"},"status":404}
```

**Issue Reasons:**
- [8.x] 2 failures in test testDeprecationIndexingCacheReset (0.5% fail rate in 407 executions)
- [8.x] 2 failures in pipeline elasticsearch-periodic-platform-support (15.4% fail rate in 13 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java
+++ b/x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java
@@ -121,7 +121,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                 List<Map<String, Object>> documents = DeprecationTestUtils.getIndexedDeprecations(client(), xOpaqueId());
                 logger.warn(documents);
                 assertThat(documents, hasSize(2));
-            }, 30, TimeUnit.SECONDS);
+            }, 45, TimeUnit.SECONDS);
         } finally {
             Response response = cleanupSettings();
             List<String> warningHeaders = getWarningHeaders(response.getHeaders());
@@ -245,7 +245,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
             var documents = DeprecationTestUtils.getIndexedDeprecations(client(), xOpaqueId);
             logger.warn(documents);
             assertThat(documents, hasSize(headerMatchers.size()));
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
     }
 
     public void testDeprecationRouteThrottling() throws Exception {
@@ -275,7 +275,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
 
     }
 
@@ -303,7 +303,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                         )
                     )
                 );
-            }, 30, TimeUnit.SECONDS);
+            }, 45, TimeUnit.SECONDS);
         } finally {
             configureWriteDeprecationLogsToIndex(null);
         }
@@ -369,7 +369,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
 
     }
 
@@ -414,7 +414,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
 
     }
 
@@ -473,7 +473,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
 
     }
 
@@ -504,7 +504,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
     }
 
     public void testReplacesInCurrentVersion() throws Exception {
@@ -534,7 +534,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
     }
 
     public void testReplacesInCompatibleVersion() throws Exception {
@@ -579,7 +579,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
     }
 
     /**
@@ -649,7 +649,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
 
     }
 
@@ -690,7 +690,7 @@ public class DeprecationHttpIT extends ESRestTestCase {
                     )
                 )
             );
-        }, 30, TimeUnit.SECONDS);
+        }, 45, TimeUnit.SECONDS);
 
     }
 

